include/dpp/integration.h

```diff
@@ -80,8 +80,10 @@ struct DPP_EXPORT integration_app {
 /**
  * @brief Represents an integration on a guild, e.g. a connection to twitch.
  */
-class DPP_EXPORT integration : public managed, public json_interface<integration> {
+class DPP_EXPORT integration : public managed<integration>, public json_interface<integration> {
 public:
+	//** Unique identifier */
+	snowflake id;
```

Comment:
I don't really like this, it's basically added tons of duplicate lines to every object to save 8 bytes of ram per instance, I think readability is more important in this case